src/components/OrgSettings/General/OrgUpdate/OrgUpdate.tsx

```
@@ -157,6 +157,26 @@ function OrgUpdate(props: InterfaceOrgUpdateProps): JSX.Element {
 
   const [isSaving, setIsSaving] = useState(false);
 
+  const handleImageChange = async (
+    e: React.ChangeEvent<HTMLInputElement>,
+  ): Promise<void> => {
+    const target = e.target as HTMLInputElement;
+    const file = target.files && target.files[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)